db/queries/createUsersAccount.js

```diff
@@ -0,0 +1,90 @@
+const { Client } = require("pg");
+require('dotenv').config();
+const bcrypt = require("bcrypt");
+
+const dbConfig = {
+  user: process.env.DB_USER,
+  host: "localhost",
+  database: process.env.DB_DATABASE,
+  password: process.env.DB_PASSWORD,
+  port: 5432,
+};
+
+const client = new Client(dbConfig);
```

Comment:
This Should Be In **db.js** file, Then Import From Anywhere If You Want To Communicate With The DB

Reply:
i tried it, but it is giving me that error and im not able to solve it with all the solution online